img/Project 3.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<!-- saved from url=(0037)http://127.0.0.1:5500/code/index.html -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Project 3</title>
+  
+  
+<link rel="stylesheet" href="./Project 3_files/mobile-style.css"><link rel="stylesheet" href="./Project 3_files/overlay.css"></head>
+
+<body data-new-gr-c-s-check-loaded="14.993.0" data-gr-ext-installed="">
+
+  <header>
+    <!-- Navigation Bar (Menu /Search /ShoppingCar /Login ) :) -->
+    <section class="nav-bar">
+      <!-- Navigation Bar (Menu  -->
+      <div class="menu-hamburguesa">
+        <a class="button" href="http://127.0.0.1:5500/code/index.html#modal">
+          <img src="./Project 3_files/icon-mobile.png"></a>
```

Comment:
Let's remember that all files as well as folders should be named with lowercase letters - `project` instead of `Project`. Let's fix that ;)

Reply:
Hi Max, I forgot to remove this file because it's no used in the project.